src/core/Akka.Tests/Pattern/BackoffSupervisorSpec.cs

```diff
@@ -90,15 +91,30 @@ public static Props Props(IActorRef probe)
         private IActorRef Create(BackoffOptions options) => Sys.ActorOf(BackoffSupervisor.Props(options));
         #endregion
 
-        [Fact(Skip = "Racy on Azure DevOps")]
+        [Fact]
         public void BackoffSupervisor_must_start_child_again_when_it_stops_when_using_Backoff_OnStop()
         {
-            var supervisor = Create(OnStopOptions());
-            supervisor.Tell(BackoffSupervisor.GetCurrentChild.Instance);
-            var c1 = ExpectMsg<BackoffSupervisor.CurrentChild>().Ref;
+            var supervisor = Create(OnStopOptions().WithManualReset());
+
+            IActorRef c1 = null;
+            AwaitCondition(() =>
+            {
+                supervisor.Tell(BackoffSupervisor.GetCurrentChild.Instance);
+                c1 = ExpectMsg<BackoffSupervisor.CurrentChild>().Ref;
+                return !c1.IsNobody();
+            });
+            c1.Should().NotBeNull();
+
             Watch(c1);
             c1.Tell(PoisonPill.Instance);
-            ExpectTerminated(c1);
+            ExpectMsg<Terminated>().ActorRef.Should().Be(c1);
+
+            AwaitAssert(() =>
```

Comment:
There is no guarantee that the supervisor will observe the child termination immediately. We know that it should, sometime in the very near future, so we need to await this.